rekor-rs/src/models/log_entry.rs

```diff
@@ -0,0 +1,69 @@
+// Design a struct that stores the response returned by Rekor after making a new entry
+
+/// Stores the response returned by Rekor after making a new entry
+// get rid of default
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LogEntry {
+    // native uuid object maybe?
```

Comment:
I'd say it is fine, in this case the struct is less a "collection of data" and more like a wrapper so that you can define specific methods on it like `is_valid()`. However, I'm not sure what I commented on here because I don't see `uuid` hanging out as a bunch of numbers...

Reply:
Oh wait, I see it as a String. : )